samples/apps/spring-petclinic/init-database.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if ! [[ -d "$SERVICE_BINDING_ROOT" ]]; then
+    echo "SERVICE_BINDING_ROOT variable not set..."
+    sleep infinity
+else
+    BINDING=${SERVICE_BINDING_ROOT}/spring-petclinic
+
+    if ! [[ -d "$BINDING" ]]; then
+        echo "No binding data found - $BINDING directory is missing..."
+        sleep infinity
+    else
+        export PG_HOST=${PG_HOST:-$(cat $BINDING/host)}
+        export PG_PORT=${PG_PORT:-$(cat $BINDING/port)}
+        export PG_NAME=${PG_NAME:-$(cat $BINDING/database)}
+        export PG_USER=${PG_USER:-$(cat $BINDING/username)}
+        export PGPASSWORD=${PGPASSWORD:-$(cat $BINDING/password)}
+
+        if [[ -z "$(psql -h $PG_HOST -p $PG_PORT -U "$PG_USER" "$PG_NAME" -c "SELECT tablename FROM pg_catalog.pg_tables" | grep pets)" ]]; then
+            curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/initDB.sql
```

Comment:
if we really need to keep this init container (see my comment about auto schema creation above), we should download `.sql` files during container build process and have them in container ready, or even better commit them into our repo.. because the version on `spring-petclinic-rest` master branch can change over time and break us at the runtime.

Reply:
the init container is not needed, after all. The only reason for it was that the standard way of initializing db via JPA failed when combined with the autoconfiguration via spring-cloud-bindings and when I wanted the application to run even without the DB... I dropped the idea based on the feedback (that the application is expected to crash with no db) and so now the standard way from spring works and therefore the init container is not necessary